ddtrace/_trace/trace_handlers.py

```diff
@@ -691,6 +691,28 @@ def _on_botocore_patched_bedrock_api_call_success(ctx, reqid, latency, input_tok
     span.set_tag_str("bedrock.usage.completion_tokens", output_token_count)
 
 
+def _on_rq_queue_enqueue_job_is_async(ctx, job):
+    span = ctx[ctx["call_key"]]
+    HTTPPropagator.inject(span.context, job.meta)
+
+
+def _on_rq_after_perform_job(ctx, job):
+    """sets job.status and job.origin span tags after job is performed"""
+    # get_status() returns None when ttl=0
+    span = ctx[ctx["call_key"]]
+    span.set_tag_str("job.status", job.get_status() or "None")
+    span.set_tag_str("job.origin", job.origin)
+    if job.is_failed:
+        span.error = 1
+
+
+def _on_rq_end_of_traced_method_in_fork(ctx, job):
```

Comment:
Again can we make this function name more general. We can potentially reuse this function in other integrations